.github/workflows/cache_data.yaml

```diff
@@ -33,7 +33,8 @@ jobs:
           gmt which -Ga @earth_relief_10m_p @earth_relief_10m_g \
                         @earth_relief_30m_p @earth_relief_30m_g \
                         @earth_relief_01d_p @earth_relief_01d_g \
-                        @earth_relief_05m_p @earth_relief_05m_g
+                        @earth_relief_05m_p @earth_relief_05m_g \
+                        @N35E135.earth_relief_03s_g.nc
```

Comment:
:confused: I even didn't know that we can use `gmt which -Ga @N35E135.earth_relief_03s_g.nc`  to download a single tile.